### PR TITLE
Fix: CRW-4120 Removing command to delete 'native-keymap' module in sync script and enabling pulp content sets to resolve libsecret & libxkbfile as rpm

### PR DESCRIPTION
**Pull request by mdrafiur**
code_3.x build fails on x86_64, s390x and ppc64le due to the following 2 reasons:

1. Failed to find module '**native-keymap**'. There was a command in the script used for syncing with upstream project which deletes the 'native-keymap' module. The command has been removed from the sync script.

2. The build requires 2 packages (**libsecret** & **libxkbfile**) to be installed into the image. The RPMs are not available at the default ubi8 yum repositories. Therefore, pulp content sets is enabled to install libsecret & libxkbfile as rpm.

### CODE REVIEW

File: devspaces-code/build/dockerfiles/content_sets_pulp.repo [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+name=rhel-8-baseos-rpms-pulp
+baseurl=http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/$basearch/baseos/os
+enabled=1
+gpgcheck=0
+
+[rhel-8-for-codeready-builder-rpms-pulp]
+name=rhel-8-codeready-builder-rpms-pulp
+baseurl=http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/$basearch/codeready-builder/os
+enabled=1
+gpgcheck=0
```
Comment (ArvinB) [inline devspaces-code/build/dockerfiles/content_sets_pulp.repo:17]: Downstream how would someone access these repo's outside of RH?

File: devspaces-code/build/dockerfiles/libc-content-provider.Dockerfile [hunk 2]
```diff
@@ -17,28 +17,16 @@ USER root
 ARG GITHUB_TOKEN=''
 ENV GITHUB_TOKEN=$GITHUB_TOKEN
 
+# Enable pulp content sets to resolve libsecret & libxkbfile as rpm
+COPY ./build/dockerfiles/content_sets_pulp.repo /etc/yum.repos.d/
```
Comment (nickboldt) [inline devspaces-code/build/dockerfiles/libc-content-provider.Dockerfile:21]: you don't need this downstream but not sure if it will break anything tbh.

**Comment by nickboldt** [comment]
Please link to the scratch build in which you validated this change. I don't see it at https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/DS_CI/job/get-sources-rhpkg-container-build_3.x/ so you must have run on commandline.

**Comment by nickboldt** [comment]
building in https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/DS_CI/job/code_3.x/501/

**Pull request merged**